docs/how-to/create-template.md

```diff
@@ -0,0 +1,33 @@
+---
+parent: How To
+nav_order: 3
+title: Create Template
+---
+
+# Create a Modular Template
+
+To create a [Modular Template](../concepts/templates.md), start with any Modular
+package type for which you want to create a template for (App/Package/etc). The
+package's contents will be copied into any new package created using the
+template.
+
+To convert the package into a template, make the following changes to the
+package's package.json:
+
+- Change `"modular" "type"` field to `"template"`
+- Add `"templateType"` field to `"modular"` with the package's Modular type.
```

Comment:
Add `modular.templateType` with the desired target package type.

(we call it "target package" in the templates page)